### DISCUSSION TITLE
关于开源面对面品牌保护的问题

### DISCUSSION THREAD

**Discussion by LinuxSuRen** [opensource-f2f/episode#220] [Ideas]
自成立起，开源面对面已经接近两年的时间，过程中得到了不少开源爱好者的支持和肯定，参与到节目的主持人也是希望能够长期地将节目进行下去。从长远角度考虑，品牌必然是一个无法绕开的话题，为了保护《开源面对面》这个品牌不被随意、潜在恶意地使用，申请注册商标可能是一个不错的选择。

本着公开、透明的原则，特此将相关想法记录再次，任何有兴趣参与到开源面对面品牌保护的小伙伴都可以在下面留言讨论。

做过一些调查后，我依然有一些困惑，包括且不限于：

* 是否需要同时申请文字、图片的商标，具体保护的细节如何
* 商标保护的行业、领域等细则该如何选择
* 目前开源面对面没有法律实体，商标是否可以落到个人上，应该落到谁身上
* 社区目前没有公共可用资金用于支付相关费用（[爱发电](https://afdian.net/a/opensource-f2f)平台上有少量的捐赠）

**Comment by MikeyWei** [comment]
> 自成立起，开源面对面已经接近两年的时间，过程中得到了不少开源爱好者的支持和肯定，参与到节目的主持人也是希望能够长期地将节目进行下去。从长远角度考虑，品牌必然是一个无法绕开的话题，为了保护《开源面对面》这个品牌不被随意、潜在恶意地使用，申请注册商标可能是一个不错的选择。
> 
> 本着公开、透明的原则，特此将相关想法记录再次，任何有兴趣参与到开源面对面品牌保护的小伙伴都可以在下面留言讨论。
> 
> 做过一些调查后，我依然有一些困惑，包括且不限于：
> 
>     * 是否需要同时申请文字、图片的商标，具体保护的细节如何
> 
>     * 商标保护的行业、领域等细则该如何选择
> 
>     * 目前开源面对面没有法律实体，商标是否可以落到个人上，应该落到谁身上
> 
>     * 社区目前没有公共可用资金用于支付相关费用（[爱发电](https://afdian.net/a/opensource-f2f)平台上有少量的捐赠）

个人（自然人）申请商标至少需要个体营业执照，否则是不能申请的。按照播客的内容属性，需要申请的商标大类应该包含38和41。

个人申请商标注册需要准备以下材料：
* 提供清晰的商标图样，并且列出需要注册的商标类别以及商品或者服务。
* 提供注册申请主体的资料：个人提供身份证复印件（港澳台外籍个人可以用护照）；个体户提供营业执照复印件。
* 提供商标注册申请表，由申请人签字盖章确认。

商标注册流程包括：商标近似查询 → 形式审查（提交申请受理）→ 实质审查 → 商标公告 → 领取证书；可以选择自行办理或委托代理机构办理。
商标注册所需时间：至少 3 个月审核期 + 3 个月公示期。

**Comment by LinuxSuRen** [comment]
翻到一个采用相同名字的节目（栏目） https://developer.aliyun.com/topic/openpolardb_talk

**Reply by MikeyWei** [reply]
除非开源面对面播客有了很强的品牌效应和法律支持，否则这种情况无法避免。就当是另一种搭便车好了。

**Reply by LinuxSuRen** [reply]
确实是的。不过，看到相关信息，在这里也算是做一种记录。的确这块需要相应的法律支持才可以落地。
